modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultGenerator.java

```diff
@@ -1115,6 +1099,40 @@ private static String generateParameterId(Parameter parameter) {
         return operations;
     }
 
+    private Map<String,String> getAllImportsMapppings(Set<String> allImports){
```

Comment:
Can you please add docstrings to explain what these new (private) functions do (e.g. avoid duplicates)?

We want to document all methods in the default codegen/generator class to make code easier to understand for new contributors. (I do not expect you to document all methods missing the docstring. The community can help with separate PRs)

Reply:
I added some doc and used a Set to better represent that the entries are unique for the other method.